### ISSUE TITLE
Deleting a folder

### ISSUE THREAD

**Issue by terencedignon** [koshigoe/brick_ftp#60]
I apologize for all the issues.  I've tried my best to delete a folder through the Client API but cannot find a way.

The only success I've had is through the API directly doing the following
```
BrickFTP::API::File.new(path: 'development/companies2/bcd5fdf9-f488-47a7-a502-b1c41f35f264').destroy(recursive: true)
```
Do you know if there is way to accomplish this through the Client?  If busy I can put together some code.

**Comment by koshigoe** [comment]
I refined `BrickFTP::Client#{update,delete}_*` methods to specify resource as identifier. (#65)
And I released v0.5.0.

e.g. Remove folder recursively.

```ruby
BrickFTP::Client.new.delete_file('path/to/something', recursive: true)
```

(Sorry about named file not folder. The `#delete_file` method can remove file and folder.)

**Comment by terencedignon** [comment]
Thank you! This is great.
